tests/sentry/workflow_engine/endpoints/test_project_detector_index.py

```diff
@@ -55,22 +59,30 @@ def setUp(self):
         self.valid_data = {
             "name": "Test Detector",
             "detectorType": MetricAlertFire.slug,
-            "dataSource": {
-                "queryType": SnubaQuery.Type.ERROR.value,
-                "dataset": Dataset.Events.name.lower(),
-                "query": "test query",
-                "aggregate": "count()",
-                "timeWindow": 60,
-                "environment": self.environment.name,
-                "eventTypes": [SnubaQueryEventType.EventType.ERROR.name.lower()],
-            },
-            "dataConditions": [
+            "dataSources": [
```

Comment:
🤔 - i saw this in the ui doc - but idk if we can actually have multiple data sources feed into a single detector. is there a specific use case you had in mind? cc @natemoo-re

Reply:
nope, no specific use case in mind! i may have assumed the one:many relationship meant we needed to support this (now or in the future) but happy to keep it flattened on a top-level `dataSource` key.